docs/adapters/fauna.md

```diff
@@ -11,7 +11,11 @@ You can find the Fauna schema and seed information in the docs at [next-auth.js.
 
 ## Getting Started
 
-1. Install `next-auth` and `@next-auth/fauna-adapter`
+:::warning
+When using the **NextAuth v4 beta**, please make sure to use the `next` tagged version of your adapter. For more info on adapter changes, see [the migration docs](/getting-started/upgrade-v4#adapters)
+:::
+
+1. Install `next-auth` and `@next-auth/fauna-adapter@next`
```

Comment:
Can we do `1. Install `next-auth@next` and `@next-auth/fauna-adapter@next`? The inconsistent usage might confuse someone not paying attention that it's intended to install next-auth as non-next.